libheif/uncompressed_image.cc

```diff
@@ -25,38 +25,55 @@
 #include <map>
 #include <iostream>
 
+#include "libheif/heif.h"
 #include "uncompressed_image.h"
 
 
-enum heif_component_type
+enum uncompressed_component_type
```

Comment:
I think we should keep the old name with the `heif_` prefix because I expect that this enum will eventually be moved to the public API where we need the prefix. Of course, we could postpone the prefix until then, but I guess it will be sooner than later anyway.

Reply:
I had hoped to hide the detail of the uncompressed codec behind the existing API. No problem with changing the names back though.

Reply:
Yes, I understand that, but it appears that there will be many more components (UV, X-Ray, even though they apparently have been removed from the latest draft) so that we will have to expose the components enum to the user in some way to let them attach such images. We can postpone the details until later.